tests/components/tibber/test_statistics.py

```diff
@@ -39,7 +39,7 @@ async def test_async_setup_entry(recorder_mock: Recorder, hass: HomeAssistant) -
             {statistic_id},
             "hour",
             None,
-            {"start", "state", "mean", "min", "max", "last_reset", "sum"},
+            {"state", "mean", "min", "max", "last_reset", "sum"},
```

Comment:
`start` was not a valid value here

`types: set[Literal["change", "last_reset", "max", "mean", "min", "state", "sum"]],`

We could filter it, but its probably ok to blow up on invalid types and this doesn't actually happen in the prod call